client/src/components/content/ReviewForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react"
+
+import ErrorList from "../layout/ErrorList.js"
+import translateServerErrors from "../../services/translateServerErrors.js"
+
+const ReviewForm = props => {
+  const [newReview, setNewReview] = useState('')
+  const [errors, setErrors] = useState([])
+
+  const addNewReview = async (event) => {
+    event.preventDefault()
+    const { characterId } = props
+    const reviewBody = {
+      content: newReview
+    }
+
+    try {
+      const response = await fetch(`/api/v1/characters/${characterId}/reviews`, {
+        method: 'POST',
+        headers: new Headers({
+          'Content-Type': 'application/json'
+        }),
+        body: JSON.stringify(reviewBody)
+      })
+      if (!response.ok) {
+        if (response.status === 422) {
+          const body = await response.json()
+          const newErrors = translateServerErrors(body.errors)
+          return setErrors(newErrors)
+        } else {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error(errorMessage)
+          throw (error)
+        }
+      } else {
+        setErrors([])
+        setNewReview("")
+        props.getReviews()
```

Comment:
I totally get why you did this, but it requires multiple fetches. just grab the new review when on the backend from the POST

you may also need to move this whole function into the Show component to better update that state